src/app/error-check/error-check.component.ts

```diff
@@ -0,0 +1,28 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-error-check',
+  templateUrl: './error-check.component.html',
+  styleUrls: ['./error-check.component.css']
+})
+export class ErrorCheckComponent implements OnInit {
+
+  constructor() { }
+
+  ngOnInit() {
+  }
+
+  checkForError(error): string {
```

Comment:
nice!

Reply:
To prevent displaying errors, when we get something unpredicted, like HTTP 500. So user won't get totally random message 'Internal server error'